### PR TITLE
task definition volume

### PR DESCRIPTION
**Pull request by jtdev-gerald**
## what
volume argument not supported in task definition

## why
I need to mount my container on task definition volume to support lookup on docker Labels

## references
https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ecs_task_definition#volume

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -35,6 +35,25 @@ locals {
     }
   ] : var.mount_points
 
+  dynamic "volume" {
```
Comment (nitrocode) [inline main.tf:38]: Can't use a `dynamic` in `locals`

**Comment by nitrocode** [comment]
This repo will only give you the container definition

e.g.

```hcl
module "container_definition" {
  source  = "cloudposse/ecs-container-definition/aws"
  version = "0.58.0"

  container_name  = "name"
  container_image = "cloudposse/geodesic"
}

resource "aws_ecs_task_definition" "service" {
  container_definitions = module.container_definition.some_output

  volume {
    name      = "service-storage"
    host_path = "/ecs/service-storage"
  }
}
```

**Comment by jtdev-gerald** [comment]
That you guy that make sense to me.

**Pull request closed without merging**